frontend/src/Components/ClientProfileContent.js

```diff
@@ -338,39 +398,46 @@ class ClientProfileContent extends Component {
       }
       return (
         <React.Fragment>
-          <div>
-            <InputComponent
-              type='text'
-              label='Short Term Goal (Examples: taking a vacation, buying a car)'
-              placeholder='Type in your short term goal'
-              value={props.shorttermgoal}
-              name='shorttermgoal'
-              onChange={props.handleChange}
-              required
-            />
-          </div>
-          {/* TODO: Add inputs on button click */}
-          <Center>
-            <ToastButtonComponent tertiary label='+ Short Term Goal' />
-          </Center>
-
-          <hr />
-
-          <InputComponent
-            type='text'
-            label='Long Term Goal (Examples: buying a house, preparing for retirement)'
-            placeholder='Type in your long term goal'
-            value={props.longtermgoal}
-            name='longtermgoal'
+          <DropdownComponent
+            options={goalOptions}
+            name='goal' 
+            label='Goal 1'
+            list='goals'
+            placeholder='Type in your goal'
+            value={props.goal}
+            id="goals"
             onChange={props.handleChange}
-            required
           />
 
+          <div className='row'>
+            <div className='column'>
+              <InputComponent
+                type='number'
+                label='Dollar Amount'
+                name='dollarAmount'
+                placeholder='1,000'
+                min={0}
```

Comment:
for $ inputs, we can stick to 'step={0.01}' so this should be added here. not sure if min={0.0} is necessary since it'll say 0 anyways but you can change that if you want to as well